apps/meteor/client/views/omnichannel/agents/AgentsTable/AgentsTable.tsx

```diff
@@ -84,7 +88,16 @@ const AgentsTable = ({ reload }: { reload: MutableRefObject<() => void> }) => {
 					</GenericTableBody>
 				</GenericTable>
 			)}
-			{isSuccess && data.users.length === 0 && <GenericNoResults />}
+			{isSuccess && data?.users.length === 0 && queryHasChanged && <GenericNoResults />}
+			{isSuccess && data.users.length === 0 && !queryHasChanged && (
+				<GenericNoResults
+					icon='headset'
+					title={t('No_agents_yet')}
+					description={t('No_agents_yet_description')}
+					linkHref='https://go.rocket.chat/omnichannel-docs'
+					linkText={t('Learn_more_about_agents')}
+				/>
+			)}
```

Comment:
`question` I didn't get what is happening here.

Reply:
if the query is 'empty' or the default shows 'not agents yet, go to docs to see how to add' if the user is searching for some term and didnt find because of the filters: "not agents found"